templates/login/index.html.twig

```diff
@@ -9,6 +9,13 @@
         <div class="img-user">
             <img src="{{ asset('build/images/user.png') }}" alt="img profil">
         </div>
+        {% for label, messages in app.flashes %}
```

Comment:
I think that the flashes error message display should be in the base.html.twig, bellow the header